bump_packaging

```diff
@@ -51,7 +51,9 @@ fi
 git commit -m "Release $FULLVERSION"
 
 if [[ $PACKAGING_PR == true ]] ; then
-	git phr -m "Release $FULLVERSION" -b "$REMOTE_BRANCH"
+	# TODO: assumes a remote named whoami for the user's fork
```

Comment:
If you really want to script this and reduce the amount of assumptions and manual set-up, our team has been using this ansible role for well over a year to deploy all of the source and tooling repositories used for building hotfix RPMs: https://github.com/theforeman/forklift/pull/1580

I've been using it to manage all of the repositories in my katello devel box for a similar length of time as well.

Reply:
I'm not a fan of using VMs for those common tasks. It's always awkward when I miss my own customization.